src/js/themes/base.js

```diff
@@ -1036,6 +1040,59 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    notification: {
+      toast: {
+        container: {
+          gap: 'small',
+          elevation: 'medium',
+          round: 'small',
+        },
+      },
+      iconContainer: {
+        pad: 'small',
+        round: { size: 'small', corner: 'left' },
+      },
+      textContainer: {
+        pad: { left: 'small', right: 'medium', vertical: 'small' },
+      },
+      messageText: {
+        size: 'large',
+        weight: 'bold',
+        color: 'black',
+      },
+      bodyText: {
+        size: 'medium',
+        margin: 'none',
+        color: 'black',
+      },
+      button: {
+        margin: { left: 'medium' },
```

Comment:
We don't usually apply a margin to buttons. Would it be possible in the notification file to add a `gap` instead? between the notification text and this button?